mindocr/scheduler/dynamic_lr.py

```diff
@@ -22,6 +22,16 @@
 import math
 from bisect import bisect_right
 
+from mindspore import nn
+
+
+def dynamic_lr(lr, end_lr, epochs, step_per_epoch, by_epoch=True):
```

Comment:
we already have `polynomial_decay` in `mindocr/scheduler/scheduler_factory.py`. Isn't it the same?